lib/berbix.rb

```diff
@@ -236,4 +235,6 @@ def api_host(opts)
     end
   end
 
+  class HttpError < StandardError
```

Comment:
I have a preference for this to be a broader term than `Http`, perhaps `Berbix`. The rationale being that we may want to be more specific in the future about what errors are being thrown (`ValidationError`, `AuthenticationError`, `ServerError`, `NetworkError`, etc), but that this would ensure that it is backwards compatible with clients who only consume the top-level `BerbixError`, whereas the specificity of `HttpError` implies that it is related specifically to an http call.

Reply:
I love that idea. In fact I wrote it that way initially, but was worried that because of the way it was being invoked (in the context of the specific error where it was raised) it might be deemed to broad. I'll update it now.